tests/stormpy_interface_test.py

```diff
@@ -0,0 +1,184 @@
+# This test includes integration tests for the interfaces.stormpy module,
+# using the example from the EECI 2020 computer lab.
+# Details of these tests can be found at
+# the Stormpy computer session of EECI 2020:
+#
+# https://www.cds.caltech.edu/~murray/wiki/index.php?title=EECI_2020:_Computer_Session:_Stormpy
+#
+# All the tests here are run only if on Python 3
+#
+# * light_analysis_test() corresponds to Example 1: traffic light (analysis)
+# * compose_test() corresponds to Example 2: traffic light + vehicle (analysis)
+# * synthesis_test() corresponds to Example 4: policy synthesis
+
+import sys
+if sys.version_info.major == 2:
+    skip_test = True
+else:
+    skip_test = False
+
+import os
+import unittest
+
+if not skip_test:
+    from tulip.transys.compositions import synchronous_parallel, apply_policy
+    from tulip.interfaces import stormpy as stormpy_int
+
+# Set up paths to all the models
+model_path = os.path.join(os.path.dirname(os.path.abspath(__file__)), "models")
+light_path = os.path.join(model_path, "light.pm")
+ma_path = os.path.join(model_path, "ma.nm")
+mh_path = os.path.join(model_path, "mh.pm")
+out_model_path = os.path.join(model_path, "tmpout.nm")
```

Comment:
It is best to use [mkstemp](https://docs.python.org/3.7/library/tempfile.html#tempfile.mkstemp) to create a temporary file according to standard OS calls or to use a temporary directory fixture provided by the test runner. The latter will be [available in pytest](https://docs.pytest.org/en/stable/tmpdir.html) after we switch to it (task from issue #169).

However, there is code that follows the same pattern as here [in gr1cint_test.py](https://github.com/tulip-control/tulip-control/blob/9b9a743d1486ded5a91eba2fbbb4536559b85f5f/tests/gr1cint_test.py#L171). It has not caused trouble in practice yet, so I think we can merge the code here and possibly update to mkstemp() during work on #169

Reply:
Very nice suggestion. I didn't know about this. I'll revisit this as well as the previous comment.